action.yml

```diff
@@ -81,8 +86,20 @@ runs:
             exit 1
         fi
         git checkout -b ${{ inputs.new_branch_name }}
-        git config user.name github-actions
-        git config user.email github-actions@github.com
+
+        git_user_name=${{ inputs.git_user_name }}
+        git_user_email=${{ inputs.git_user_email }}
+
+        if [[ -z ${git_user_name} ]]; then
+            git_user_name=github-actions
+        fi
+
+        if [[ -z ${git_user_email} ]]; then
+            git_user_email=github-actions@github.com
```

Comment:
Same thing here.

We could add the `github-actions@github.com` value as default to the inputs configuration above to remove this check. What do you think?
 
**Example:**
```
    git_user_email:
        description: 'Allows setting user.email in git config'
        required: false
        default: 'github-actions@github.com'
 ```